velox/docs/functions/spark/datetime.rst

```diff
@@ -34,6 +34,12 @@ These functions support TIMESTAMP and DATE input types.
     deducted from ``start_date``.
     Supported types for ``num_days`` are: TINYINT, SMALLINT, INTEGER.
 
+.. spark:function:: date_from_unix_date(integer) -> date
+
+    Creates date from the number of days since 1970-01-01. Returns null when input is null.
+
+        SELECT date_from_unix_date(1); -- '1970-01-02'
```

Comment:
Can input be negative? Would you add an example?

date_from_unix_date(-100)

Reply:
Input can be negative, added an example.